fe/fe-core/src/test/resources/sql/optimized-plan/join.sql

```diff
@@ -198,16 +198,17 @@ RIGHT SEMI JOIN (join-predicate [14: cast = 6: t1f] post-join-predicate [null])
 [sql]
 select v1,v2,v3,v4 from t0 full outer join t1 on v1=v5 and 1>2
```

Comment:
should add new test cases to cove previous test logic

Reply:
added